lighthouse-core/lib/i18n/locales/de.json

```diff
@@ -630,7 +648,7 @@
     "message": "Für Nutzer mit langsamen Verbindungen können externe Skripts, die dynamisch über `document.write()` eingefügt werden, den Seitenaufbau um einige Sekunden verzögern. [Weitere Informationen.](https://web.dev/no-document-write)"
   },
   "lighthouse-core/audits/dobetterweb/no-document-write.js | failureTitle": {
-    "message": "Verwendet `document.write()`"
+    "message": "Avoid `document.write()`"
```

Comment:
one weird thing is that all the languages adopted the english string for the document.write `failureTitle`, which doesn't seem like the correct behavior? I would assume it would have been dropped since there wasn't a new translation

@exterkamp

Reply:
Yeah I did see these removals: 

![image](https://user-images.githubusercontent.com/39191/81243389-866eed00-8fc4-11ea-84f0-2daff1961f3b.png)

Reply:
Shane clarified that we only 'remove messages' when its not part of our source files. (not in the ctc)

We don't remove messages that are not-yet-translated. And they are deliberately populated with the fallback (en) strings.